examples/todo-backend-typescript/run.sh

```diff
@@ -1,5 +1,3 @@
 #!/bin/sh
 export FEATUREHUB_ACCEPT_BAGGAGE=true
-export FEATUREHUB_EDGE_URL=http://localhost:8903/
```

Comment:
i don't think this should be removed?

Reply:
yes, because it overrides the use of the `create-demo.sh` script back in the main repository which automatically creates the new portfolio + application + environments + features and sets their initial states and exports all of the appropriate environment variables.